channeldb/channel.go

```diff
@@ -941,6 +1054,20 @@ func (c *OpenChannel) fullSync(tx kvdb.RwTx) error {
 		return ErrChanAlreadyExists
 	}
 
+	cid := lnwire.NewChanIDFromOutPoint(&c.FundingOutpoint)
+	allOps := cid.GenPossibleOutPoints()
+	var otherOpBuf bytes.Buffer
+	for _, op := range allOps {
```

Comment:
I think this is not a good idea since we'll be getting 65k keys in this loop... It also makes the etcd STM blow up. Context: https://github.com/lightningnetwork/lnd/issues/6412

Reply:
Wouldn't a linear scan until `Get` returns `nil` work?

Reply:
Are `Get` calls in-order lexicographically?

Reply:
I meant that do we need to iterate the whole range and check if the keys are in the `opBucket`? Or could we just iterate until we find a non-existing key?

Reply:
It just calls Seek once and that's it now

Reply:
My prior approach didn't work, is etcd not capaable of handling 65k Get calls? Otherwise the only way I can think of to fix this is with a migration